app/src/main/java/org/p2p/wallet/common/di/AppScope.kt

```diff
@@ -6,5 +6,5 @@ import kotlinx.coroutines.SupervisorJob
 import kotlin.coroutines.CoroutineContext
 
 class AppScope : CoroutineScope {
-    override val coroutineContext: CoroutineContext = SupervisorJob() + Dispatchers.Main
+    override val coroutineContext: CoroutineContext = SupervisorJob() + Dispatchers.IO
```

Comment:
Not sure about this

Reply:
Me too, but it's a little bit confusing to use main thread for long term tasks,
we can discuss about it.